scripts/chief_deploy.py

```diff
@@ -97,6 +97,7 @@ def update_info(ctx):
             ctx.local("svn status")
 
         ctx.local("git rev-parse HEAD > media/revision.txt")
+        ctx.local("git show --format='%ai' > media/deploytime.txt")
```

Comment:
Let's not use git here actually, since that can be misleading. E.g. check out the difference between:

`git show -s --format=%ai 356032ab82bea4248977d255820a9a5f8652dc4f`

and

`git show -s --format=%ci 356032ab82bea4248977d255820a9a5f8652dc4f`

Both of which refer to 356032ab82bea4248977d255820a9a5f8652dc4f which because it was reviewed by me changes the time of the commit versus the time of the authoring. The times differ though and therefore we should prevent our deploy system to be dependent on this type of thing.

Instead use this to generate a UTC datetime that we can easily parse later in Python

    ctx.local("date --utc --rfc-822 > media/deploytime.txt")

Here's the code for parsing it with Python:

    from datetime import datetime
    import email.utils

    date_from_file = 'Mon, 07 Sep 2015 13:37:00 +0000'
    date_in_python = datetime.utcfromtimestamp(email.utils.mktime_tz(email.utils.parsedate_tz(date_from_file)))

This isn't beautiful but we can do this with built-in features from Python.